tools/make/ffi.toml

```diff
@@ -185,75 +150,6 @@ end
 
 # Install tasks
 
-# These tasks need a minimum version of cargo-make. Configs within cargo-make
-# task stanzas such as this allow setting the minimum version of deps (including
-# min version for cargo-make itself). But cargo-make might only be applying
-# such dep upgrades defined in the `install_crate` field after the task script
-# has run.
-[tasks.cargo-make-min-version]
```

Comment:
nit: I'd like to keep these around, being able to diplomat-install is still somewhat convenient during debugging

Reply:
But we don't need `diplomat-tool` anymore